templates/location3d.html

```diff
@@ -1,74 +1,37 @@
-{% extends 'base.html' %} 
+{% extends 'base.html' %}
+{% from 'macros.html' import table %}
 {% block title %} 
 Location Page 
 {% endblock %} 
 {% block content %}
 <h1>Location - {{ locdet["name"] }}</h1>
+{% set rack_num = "{:02d}".format(rack_id| int)|string %}
 <script>
```

Comment:
if you can try not to write js directly in a html file. create a js file that does what you want, add an extra js block to your base template, and use a _<script src="{{ static("js/abc.js")" }}_ to link it to the template.